package.json

```diff
@@ -17,13 +17,23 @@
     "reload": "^2.2.2"
   },
   "devDependencies": {
+    "eslint": "^4.5.0",
+    "eslint-config-mns-core": "^1.0.0",
+    "eslint-watch": "^3.1.2",
     "mocha": "^4.1.0",
     "node-mocks-http": "^1.6.6",
     "nodemon": "^1.14.11",
+    "sinon": "^4.4.2",
+    "chai": "^4.1.1",
+    "chai-as-promised": "^7.1.1",
+    "sinon-chai": "^2.9.0",
     "supertest": "^3.0.0"
   },
   "scripts": {
-    "start": "node server",
-    "test": "mocha"
+    "start": "node server/server",
+    "dev": "npm run lint:js && npm run test",
+    "test": "mocha ./test/test.config.js data/data.test.js utils/auth.test.js",
+    "test:server": "mocha ./test/test.config.js test/test.js",
```

Comment:
Slightly confused why you have both `test` and `test:server` but only run one of them during dev.

Reply:
one needs server atm to run need to figure it out